admin/meetings/111624-sprint-1-retrospective.md

```diff
@@ -0,0 +1,42 @@
+# 11/16/2024 - Retrospective Meeting
+
+Attendees: Tanmay, Mokshita, Kshitij, Angus, Chris, Fude, Jyotika, Rohan, Chris C
+
+
+
+## Liked
+* Everyone's participation
+* Pitch planning meeting 
+* Good Progress on the analysis tasks
+* Enthusiasm in team members
+* Everything is done on time.
+* Tasks are organized
+* Clear work seperation
+* Finish tasks on time
+* Switching to Discord
+* Great communication
+* Speed at which the meeting times are decided
+
+## Learnt
+* Github issues and actions using
+* Brownfield projects 
+* Code Quality Tools
+* Github Actions, Documentation tools
+* How to document properly
+* What's linting and there are many linting to choose
+* Documentation tools
+* Theres a world of VS code extension development
+
+## Lacked
+* Keep shorter meetings
+    * Resolution: Come up with a more specific agenda for the meetings
+* Reviewing and closing the open pull requests
+    * Resolution: Assign atleast 2 reviewers and tag them on Discord/ Mattermost. Responsibility of the reviewer to review the code at the earliest.
+* Everyone is not in sync
+    * Resolution: Team members should take the initiative and ask others about the progress. Also, post updates regularly on the progress-update channel.
```

Comment:
See above about stand-ups. You have a Progress Update channel, I suggest creating a daily thread that _everyone_ should reply to at the _start_ of each day (not end). That way if anyone is having problems or stuck on something, it's possible for people who have finished tasks to move over to assisting them.

Reply:
Yes, we will try to do it this way going forward.